Resources/Prototypes/Entities/Objects/Tools/tools.yml

```
@@ -565,6 +613,39 @@
     - AirlockGlass
     - Firelock
 
+- type: entity
+  id: RPDEmpty
+  parent: RPD
+  suffix: Empty
+  components:
+  - type: LimitedCharges
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)